src/main/java/net/sf/jabref/logic/net/URLDownload.java

```diff
@@ -236,4 +242,43 @@ protected InputStream monitorInputStream(InputStream in) {
     public String toString() {
         return "URLDownload{" + "source=" + source + '}';
     }
+
+    public void fixSSLVerification() {
+
+        LOGGER.warn("Fix SSL exception by accepting ALL certificates");
```

Comment:
This prints a message into the log file *every time* the method is called. Please reduce the log level to debug.